script/run-clang-format.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python
```

Comment:
@codebytere out of curiosity, did you write this whole wrapper from scratch, or is it derived from some existing thing? I ask because I'm wondering if we could just install some "egg" and be done with it, though I don't know if there's a precedent for that in this project. I think all the python code is just one-off scripts.